wp-content/mu-plugins/postcrafter-yoast-integration/tests/run-all-tests.php

```
@@ -1 +1,434 @@
+        delete_transient('postcrafter_seo_success_notice_shown');
+        
+        // Set up WordPress environment if needed
+        if (!function_exists('wp_get_current_user')) {
+            // Mock WordPress functions for testing
+            function wp_get_current_user() {
+                return (object) array('ID' => 1, 'user_login' => 'admin');
+            }
+        }
+        
+        if (!function_exists('current_user_can')) {
+            function current_user_can($capability) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$capability'. (Unused Code Rules)

(UnusedFormalParameter)